src/modules/integrator-guide/pages/authentication/authenticate-external/index.adoc

```diff
@@ -25,28 +24,27 @@ You can use our https://github.com/cortezaproject/corteza-vue/blob/2022.9.x/src/
 
 Access {PRODUCT_NAME} {APP_NAME_ADMIN} panel and navigate to menu:system[auth clients].
 Create a new auth client which your external application is going to use to authenticate.
-//Editor's note: Given that some _Auth_ are capitalized (lines like 36, 38, 41) and some aren't (27, 57, 68...), please consider what would be more appropriate and use that all over the text. - Or is there any difference between the usage of capitalized and uncapitalized items?
 
 [IMPORTANT]
 ====
 We advise you create a separate auth client for each application that you wish to integrate with.
 Reusing the same client may grant some integration access permissions that they should not have.
 ====
 
-.The Auth client consists of:
+.The auth client consists of:
 . a combination of a client ID and a client secret (the client ID is located in the URL address bar),
-. a list of scopes to limit access based on the used Auth client,
+. a list of scopes to limit access based on the used auth client,
 . a list of redirection URIs that can be used with authorization code grant,
 . a trusted flag to force users to confirm authentication,
-. an Auth client validity based on the date.
+. an auth client validity based on the date.
 
 [IMPORTANT]
 ====
 Make sure to select the appropriate grant type based on your use-case.
 Refer to xref:authentication/index.adoc#grant-types[here] for more details.
 ====
 
-== Configuring  External Applications
```

Comment:
Same with spaces as with blank lines above

Reply:
That was apparently a typo, I'll pay more attention to that too.